DefaultWidget.ts

```diff
@@ -361,6 +361,48 @@ const MAPPING_BN: PerLanguageMapping = new Map([
 	] ],
 ]);
 	
+const MAPPING_MN: PerLanguageMapping = new Map([
+	[ "text-classification", [
+		`Би чамд хайртай`,
+	] ],
+	[ "token-classification", [
+		`Намайг Дорж гэдэг. Би Улаанбаатарт амьдардаг.`,
+		`Намайг Ганбат гэдэг. Би Увс аймагт төрсөн.`,
+		`Манай улс таван хошуу малтай.`,
+	] ],
+	[ "question-answering", [
+		{
+			text: `Та хаана амьдардаг вэ?`,
+			context: `Намайг Дорж гэдэг. Би Улаанбаатарт амьдардаг.`,
+		},
+		{
+			text: `Таныг хэн гэдэг вэ?`,
+			context: `Намайг Дорж гэдэг. Би Улаанбаатарт амьдардаг.`,
+		},
+		{
+			text: `Миний нэрийг хэн гэдэг вэ?`,
+			context: `Намайг Ганбат гэдэг. Би Увс аймагт төрсөн.`,
+		}
+	] ],
+	[ "translation", [
+		`Намайг Дорж гэдэг. Би Улаанбаатарт амьдардаг.`,
+		`Намайг Ганбат гэдэг. Би Увс аймагт төрсөн.`,
+	] ],
+	[ "summarization", [
+		`Mongolia (/mɒnˈɡoʊliə/ (About this soundlisten), Mongolian: Монгол Улс, transcription: Mongol Uls, Traditional Mongolian: ᠮᠤᠩᠭᠤᠯ ᠤᠯᠤᠰ, transliteration: Mongγol ulus) is a landlocked country in East Asia. Its area is roughly equivalent with the historical territory of Outer Mongolia, and that term is sometimes used to refer to the current state. It is sandwiched between Russia to the north and China to the south, where it neighbours the Inner Mongolia Autonomous Region. Mongolia does not share a border with Kazakhstan, although only 37 kilometres (23 mi) separate them.`,
```

Comment:
this is in EN, is this intended?

Reply:
@julien-c 

Sorry for the bad validation.
I just corrected the initialization (which was in English, now in Mongolian)

Thank you for your help!